### PR TITLE
appIcons: Show app details menu for snap applications

### PR DESCRIPTION
**Pull request by 3v1n0**
If an application is a snap package, and the snap store is installed we can show the snap packages details as we do for other non-snap applications if GNOME Software is installed

UDENG-4316
UDENG-4700

### CODE REVIEW

File: appIcons.js [hunk 1]
```diff
@@ -1184,7 +1189,8 @@ const DockAppIconMenu = class DockAppIconMenu extends PopupMenu.PopupMenu {
             }
 
             if (Shell.AppSystem.get_default().lookup_app('org.gnome.Software.desktop') &&
```
Comment (sergio-costas) [inline appIcons.js:1191]: Does this still apply? I mean: with the new Snap Store, there is nothing with `org.gnome.Software` at/in `/var/lib/snapd/desktop/applications`...

**Review by sergio-costas — COMMENTED** [review]
Just a question...

File: appIcons.js (hunk 1 above)
Comment (3v1n0) [reply appIcons.js:1191]: That's the code for non-snaps, dash-to-dock it's not an ubuntu project so we definitely support also default GNOME installations where GNOME software is installed.

Comment (sergio-costas) [reply appIcons.js:1191]: Oh, ok. Understood.

**Pull request merged**